source/solutions-library.txt

```diff
@@ -56,7 +56,17 @@ kick-start their projects.
             :icon-alt: Atlas general_content_play icon
 
             Find out how AI is being used in renewable energy by leveraging MongoDB 
-            Atlas Vector Search to drive efficiency through real-time, audio diagnostics. 
+            Atlas Vector Search to drive efficiency through real-time, audio diagnostics.
+
+         .. card::
+            :headline: Building an IoT data hub for smart manufacturing
```

Comment:
[Minor]

For consistency with other cards, can we use title case in the headline rather than sentence case?